src/Components/common/BigTile/index.js

```diff
@@ -0,0 +1,62 @@
+import {
+  Container,
+  TileImage,
+  TileContainer,
+  TileTitle,
+  TileYear,
+  TileSubHeaderContainer,
+  TileSubHeader,
+  TileSubHeaderTitle,
+  TileSubHeaderInformation,
+  TileGenreContainer,
+  TileGenre,
+  TileVotesContainer,
+  TileRatesContainer,
+  TileStarIcon,
+  TileActualRating,
+  TileMaxRating,
+  TileVoteAmount,
+  TileArticle,
+} from "./styled";
+import poster from "../../../images/poster.png";
+
+export const BigTile = () => (
+  <Container>
+    <TileImage src={poster} />
```

Comment:
Uważam że elementy zawarte w komponencie nie muszą być nazywane od nazw komponentu tylko porostu czym są w całej strukturze, czyli jak masz <TileImage  /> to powinno wystarczyć samo <Image  /> itd.